superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -426,14 +426,7 @@ export default class FilterableTable extends PureComponent<
   }) {
     const columnKey = this.props.orderedColumnKeys[columnIndex];
     const cellData = this.list[rowIndex][columnKey];
-    const content =
-      cellData === null ? (
-        <i className="text-muted">
-          {this.getCellContent({ cellData, columnKey })}
-        </i>
-      ) : (
-        this.getCellContent({ cellData, columnKey })
-      );
```

Comment:
do we still need this if the table has more than 50 columns? (i.e., renders a grid?)

Reply:
Maybe we could just move the logic to `getCellContent`?

Reply:
Oops, good catch! Added it back in [`this commit`](https://github.com/apache/superset/pull/17385/commits/2f152047ea98eac08fb482a0f5834fd98ced20e7).

My initial attempt was to add this logic to `getCellContent`, but that function needs to get just the text content of each cell, it doesn't like components. It uses the text to get the width of the columns. I tried working around the function to get it to accept elements too but I couldn't find a way, since it needs to output just a string of what's inside the cell for sizing purposes.

Reply:
Can you confirm that it also works with more than 50 columns?

Reply:
I've updated the description with screenshots for table and grid